### PR TITLE
DEVDOCS-2238: Building Channels Quick Start

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2238](https://jira.bigcommerce.com/browse/DEVDOCS-2238)

## What changed?
* Add Building Channels Quick Start and screenshots.

### CODE REVIEW

File: docs/api-docs/channels/channels-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Create a Channel Quick Start
+
+<div class="otp" id="no-index">
+
+
+### On This Page
```
Comment (traci707) [inline docs/api-docs/channels/channels-quick-start.md:6]:
```suggestion
### On this page

```

File: docs/api-docs/channels/channels-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+### On This Page
+
+- [Create a channel](#create-a-channel)
+- [Create a channel with navigation](#create-a-channel-with-navigation)
+- [Resources](#resources)
+
+</div>
+
+This article is an advanced quick start tutorial for [bigcommerce technology partners](https://www.bigcommerce.com/partners/) wishing to market their solution within [Channel Manager's](https://developer.bigcommerce.com/api-docs/channels/overview#resources) **Create Channel** flow. For an introduction to channels on BigCommerce, see [Channels Overview](https://developer.bigcommerce.com/api-docs/channels/overview).
```
Comment (traci707) [inline docs/api-docs/channels/channels-quick-start.md:14]:
```suggestion
This article is an advanced quick start tutorial for [Bigcommerce technology partners](https://www.bigcommerce.com/partners/) wishing to market their solution within [Channel Manager's](https://developer.bigcommerce.com/api-docs/channels/overview#resources) **Create Channel** flow. For an introduction to channels on BigCommerce, see [Channels Overview](https://developer.bigcommerce.com/api-docs/channels/overview).

```
```

File: docs/api-docs/channels/channels-quick-start.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+### Prerequisites
+* Experience [building apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
+* Familiarity with [Channels Toolkit](https://developer.bigcommerce.com/api-docs/channels/overview)
+
+## Create a channel
+
+At the very least, apps curated as solutions within the **Create Channel** flow must create a basic Channel with a reference to their BigCommerce **App ID**. This enables them to show in the Channel Manager as managing a specific channel.
```
Comment (traci707) [inline docs/api-docs/channels/channels-quick-start.md]:
```suggestion
At the very least, apps curated as solutions within the **Create Channel** flow must create a basic channel with reference to their BigCommerce **App ID**. This enables them to show in the Channel Manager as managing a specific channel.

```

File: docs/api-docs/channels/channels-quick-start.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+> * For a list of accepted values for `type` and `platform`, see [Channels API Reference](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api#platform)
+
+</div>
+</div>
+</div>
+
+## Create a channel with navigation
+
+It is recommended that apps also create a navigation section to better guide the merchant within the BigCommerce native control panel.
+
```
Comment (traci707) [inline docs/api-docs/channels/channels-quick-start.md]:
```suggestion
We recommended that apps create a navigation section to better guide the merchant within the BigCommerce native control panel.


```

**Review by traci707 — DISMISSED** [review]

File: docs/api-docs/channels/channels-quick-start.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+### On this page
+
+
+- [Create a channel](#create-a-channel)
+- [Create a channel with navigation](#create-a-channel-with-navigation)
+- [Resources](#resources)
+
+</div>
+
+This article is an advanced quick start tutorial for [Bigcommerce technology partners](https://www.bigcommerce.com/partners/) wishing to market their solution within [Channel Manager's](https://developer.bigcommerce.com/api-docs/channels/overview#resources) **Create Channel** flow. For an introduction to channels on BigCommerce, see [Channels Overview](https://developer.bigcommerce.com/api-docs/channels/overview).
```
Comment (ektravel) [inline docs/api-docs/channels/channels-quick-start.md:15]: Could this content also be relevant for Agency Partners?  
```suggestion
This article is an advanced quick start tutorial for [BigCommerce partners](https://www.bigcommerce.com/partners/) wishing to market their solution within [Channel Manager's](https://developer.bigcommerce.com/api-docs/channels/overview#resources) **Create Channel** flow. For an introduction to channels on BigCommerce, see [Channels Overview](https://developer.bigcommerce.com/api-docs/channels/overview).

```

File: docs/api-docs/channels/channels-quick-start.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+
+### Prerequisites
```
Comment (ektravel) [inline docs/api-docs/channels/channels-quick-start.md:18]: Currently, the Prerequisites section is not included in On this page. Other tutorials have it called out in On this page. For consistency, I propose to add it to On this page.

File: docs/api-docs/channels/channels-quick-start.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+* Experience [building apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
+* Familiarity with [Channels Toolkit](https://developer.bigcommerce.com/api-docs/channels/overview)
+
+## Create a channel
+
+Apps curated as solutions within the **Create Channel** flow must create a basic channel with a reference to their BigCommerce [App ID](https://developer.bigcommerce.com/api-docs/apps/tutorials/id). This enables them to be shown in the [Channel Manager](https://developer.bigcommerce.com/api-docs/channels/overview) as managing a specific channel.
+
+![channels-quick-start-01](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/channels-quick-start-01.png "channels-quick-start-01.png")
+
+And, it also enables an improved native look and feel for the merchant using the app.
```
Comment (ektravel) [inline docs/api-docs/channels/channels-quick-start.md:28]:
```suggestion
Additionally, it enables an improved native look and feel for the merchant using the app.

```

File: docs/api-docs/channels/channels-quick-start.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#requestrunner)
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> * For a list of accepted values for `type` and `platform`, see [Channels API Reference](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api#platform).
```
Comment (ektravel) [inline docs/api-docs/channels/channels-quick-start.md:65]:
```suggestion
> * For a list of accepted `type` and `platform` values, see [Channels API Reference](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api#platform).

```

File: docs/api-docs/channels/channels-quick-start.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+> * For instructions on finding your app's ID, see [Find and App's ID](https://developer.bigcommerce.com/api-docs/apps/tutorials/id).
+
+</div>
+</div>
+</div>
+
+## Create a channel with navigation
+
+We recommend that apps also create navigation sections to better integrate the app's interface within the BigCommerce control panel.
```
Comment (ektravel) [inline docs/api-docs/channels/channels-quick-start.md:74]:
```suggestion
We recommend that apps create navigation sections to better integrate the app's interface within the BigCommerce control panel.

```

File: docs/api-docs/channels/channels-quick-start.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#requestrunner)
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> * For additional information on [channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels) `config_meta` properties, see the [create a channel request body schema](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#request-body).
```
Comment (ektravel) [inline docs/api-docs/channels/channels-quick-start.md:124]:
```suggestion
> * For additional information on the `config_meta` object properties, see the [create a channel request body schema](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#request-body).

```

**Review by ektravel — COMMENTED** [review]
Added my suggestions.

File: docs/api-docs/channels/channels-quick-start.md (hunk 6 above)
Comment (aglensmith) [reply docs/api-docs/channels/channels-quick-start.md:18]: Its not suppose to be included.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**